prompterator/main.py

```diff
@@ -470,6 +470,26 @@ def set_up_prompt_attrs_area(st_container):
             height=c.DATA_POINT_TEXT_AREA_HEIGHT,
         )
 
+        system_prompt = env.from_string(st.session_state.system_prompt).render(**st.session_state.row.to_dict())
+        user_prompt = env.from_string(st.session_state.user_prompt).render(**st.session_state.row.to_dict())
+        # Show the rendered System and User prompts
+        col1, col2 = st.columns([1, 1])
```

Comment:
@samsucik @sweco @danielajass I am honestly not sure if I like it this way, it does bloat the UI quite a bit. It did seem much better than not seeing the rendered version at all :)